lib/api/v1/queries/keywords_test.go

```diff
@@ -0,0 +1,114 @@
+package queries_test
+
+import (
+	"github.com/markgravity/golang-ic/lib/api/v1/queries"
+	"github.com/markgravity/golang-ic/test"
+	"github.com/markgravity/golang-ic/test/fabricators"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("KeywordsQuery", func() {
+	Describe("Where", func() {
+		AfterEach(func() {
+			test.CleanUpDatabase()
+		})
+
+		Context("Given VALID params", func() {
```

Comment:
Let's just clean up a bit these context:

- given valid param
  - given Limit
  - given Offset
- given incorrect user

(so the happy path cases come first)